src/model/main.py

```diff
@@ -3,6 +3,8 @@
 
 from src.model.model import Model
 
+SAVE_MODEL_PATH = 'src/model/model_new.pickle'
```

Comment:
I don't think this should be inside `src` as it's not part of the code - perhaps create a `./model` directory in the root instead